### DISCUSSION TITLE
Data fetching on server/client

### DISCUSSION THREAD

**Discussion by Gringo675** [nuxt/nuxt#17835] [Questions]
Hi, please help understand this.
As I see, on the initial page load, data fetching with useFetch or $fetch makes two requests to the api: first from the server, and second from the client. Can't figure out, if there is a way not to do the second, from the client?
Let's say I've dynamic route to the product page like '/product/[product_id]'. And file [product_id].vue should take data of each product from the database via useFetch('api/getProduct'). So, when we are loading the page, we have two identical requests to api/getProduct, which increases the load on the server. How to do only one? Or if there is another way to take that data? Thanks.

**Comment by chenyuncai** [comment]
Me too. Someone said it is about the favicon missing, but it‘s not.

Client run the setup code and aways run the useAsyncData.refresh, I think it is the 'nuxt.payload.serverRendered' got a wrong status value in asyncData.mjs.

**Reply by lucassimines** [reply]
With the last update, when entering in a page that uses `useFetch`, the list appears and then starts loading, it was to be the reverse.. shows the loader and then the list.

**Reply by Gringo675** [reply]
> Me too. Someone said it is about the favicon missing, but it‘s not.
> 
> Client run the setup code and aways run the useAsyncData.refresh, I think it is the 'nuxt.payload.serverRendered' got a wrong status value in asyncData.mjs.

So it is incorrect behavior? I just started to learn Nuxt, and don't know how in should be.
